### ISSUE TITLE
How do I had some sort of marker ontop of the picker?

### ISSUE THREAD

**Issue by JordanRoPo** [gizmosachin/ColorSlider#8]
If i wanted to add a bar, so the user could visually see where they were pressing a bit more clearly, where in the code would I add this? i've tried adding it into the commoninit as well as the drawRect, however the UIView i attempt to place on top is never visible. Only if I add it in the VC where the colorSlider is instantiated, can I do this, but this feels like bad approach. I don't want it like the example, where a color preview circle appears outside of the bar.

**Comment by gizmosachin** [comment]
Please see the implementation of the preview view in `ColorSlider.swift` for an example of how to add custom markers like you described.

**Comment by JordanRoPo** [comment]
your example is nothing like what i described, which is exactly why i posted this. I stated I don't want it to be created inside the VC, i would like to contain it within the color slider as to reduce the amount of copy code that could be required if I reused this object some place else. So if I want to add a uiview ontop of the control, how can i do so? every time I try to insert the view, it never appears on top of the control.
